src/assets/custom/css/custom-softmod-feature.scss

```diff
@@ -0,0 +1,751 @@
+  }
+  .case-studies__case-text-container {
+    padding: 1rem 1rem 5.5rem;
+    background: white;
+    position: relative;
+    height: 100%;
+  }
+  .case-studies__case-text-container > .btn {
+    position: absolute;
+    left: 50%;
+    -webkit-transform: translate(-50%);
+        -ms-transform: translate(-50%);
+            transform: translate(-50%);
+    bottom: 1rem;
+  }
+  .case-studies__case-text-container > .link-button {
+    white-space: nowrap;
+    padding: 6% 10%;
+    margin-top: 10%;
+    display: inline-block;
+  }
+  @media (max-width: 1199px) {
+      .wrap-break {
+          width: 100%;
+          display: block;
+      }
+      .case-studies__cases-wrapper {
+        -webkit-box-pack: center;
+            -ms-flex-pack: center;
+                justify-content: center;
+      }
+      .case-studies__case {
+        margin: 0 1.5rem 3rem;
+        width: 80%;
+      }
+  }
+  @media (max-width: 999px) {
+    .case-studies {
+        padding-bottom: 3rem;
+      }
+  }
+  @media (max-width: 700px) {
+    .case-studies {
+      padding-top: 3rem;
+    }
+    .case-studies:before {
+      right: 0;
+    }
+  }
+  @media (max-width: 587px) {
+    .case-studies__case {
+      text-align: center;
+      max-width: 100%;
+      margin-left: 0;
+      margin-right: 0;
+    }
+    .case-studies__case .btn {
+        width: 90%;
+    }
+  }
+
+/********************************
+    Module - CTA slice
+********************************/
+
+.sm-cta-slice {
+    position: absolute !important;
+    width: 100%;
+    z-index: 5;
+    line-height: 34px;
+
+    .sm-icon--cta {
+        width: 30px;
+        height: auto;
+        float: left;
+        margin-right: 15px;
+    }
+
+    a {
+        border-bottom: 1px solid #FFFFFF;
+
+        &:hover {
+        text-decoration: none;
+        }
+    }
+ }
+
+.sm-icon--cta {
+    width: 30px;
+    height: auto;
+    float: left;
+    margin-right: 20px;
+    fill: transparent;
+    stroke: #FFFFFF;
+    stroke-width: 5px;
+ }
+
+/********************************
+    Module - CTA block
+********************************/
+
+.sm-cta-block {
+    width: 100%;
+    min-height: 400px;
+
+    .sm-cta-block__lh {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-cta.jpg');
+        background-size: cover;
+        background-position: center;
+        display: none;
+    }
+
+    .sm-cta-block__rh {
+        display: flex;
+        align-items: center;
+        text-align: center;
+        justify-content: center;
+        width: 100%;
+        height: 400px;
+        padding: 0 1.5rem;
+
+        div {
+            max-width: 500px;
+        }
+    }
+}
+
+@media (min-width: 700px) {
+
+    .sm-cta-block {
+
+        .sm-cta-block__lh {
+            display: block!important;
+        }
+
+        .sm-cta-block__lh,
+        .sm-cta-block__rh {
+            width: 50%;
+            height: 400px;
+            float: left;
+        }
+    }
+}
```

Comment:
This file is huge. One day let's bring this repo up to data so we can have smaller CSS files and bundle/include them at build time?